### PR TITLE
Project_components

### PR DESCRIPTION
**Pull request by emmanuellekamwa**
In this milestone, I have:
- Created a new directory called `components`,
- Created a new js file inside the directory,
- Used class-based components,
- Added some style to the app

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import '../index.css';
+import ButtonKey from './buttonKey';
+import Display from './Display';
+
+function App() {
+  return (
+    <>
+      <Display />
```
Comment (henatan99) [inline src/components/App.js]: - [ ] You need t0 use class-based components as per the project's requirement. The objective of this milestone is to practice React class components. Kindly refactor this component to a class-based component. :pray:

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display() {
+  return (
+    <div />
```
Comment (henatan99) [inline src/components/Display.js]: - [ ] You need t0 use class-based components as per the project's requirement. The objective of this milestone is to practice React class components. Kindly refactor this component to a class-based component. :pray:

File: src/components/Display.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  );
+}
+
+Display.prototype = {
```
Comment (henatan99) [inline src/components/Display.js]: - [ ] I think you meant `propTypes` here. `Kindly fix the typo. :pray: As a result of this the `Display` component is not rendered in the parent component. 

image

File: src/components/buttonKey.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+function ButtonKey() {
+  return (
+    <div className="calculator">
+      <div className="button-key">
+        <button data="AC" className="btn" type="button">AC</button>
+        <button data="+/-" className="btn" type="button">+/-</button>
+        <button data="%" className="btn" type="button">%</button>
+        <button data="÷" className="btn-operator" type="button">÷</button>
```
Comment (henatan99) [inline src/components/buttonKey.js]: - [ ] You need t0 use class-based components as per the project's requirement. The objective of this milestone is to practice React class components. Kindly refactor this component to a class-based component. :pray:

File: src/components/buttons.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+function Button({ name }) {
+  return (
+    <>
+      <button type="button">{name}</button>
+    </>
```
Comment (henatan99) [inline src/components/buttons.js]: - [ ]  You need t0 use class-based components as per the project's requirement. The objective of this milestone is to practice React class components. Kindly refactor this component to a class-based component. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @emmanuellekamwa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] Even though the default readme gives some general info about react app, you should update it customized to your project with minimum possible description. I recommend you use the [Microverse readme template[(https://github.com/microverseinc/readme-template).

_Check the comments under the review._

## Optional suggestions
- [ ] OPTIONAL: It is better if you commit more frequently and make your commits more descriptive. It helps track your changes easily and it makes your repo professional. Kindly mind it in your future changes. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @emmanuellekamwa, 

Good job on implementing the suggested changes. You've implemented the class-based component on your project. Well Done.

Your project is complete!  There is nothing else to say other than... it's time to merge it :detective: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**